src/fastgltf.cpp

```diff
@@ -390,6 +401,15 @@ std::pair<fg::Error, fg::DataSource> fg::glTF::decodeDataUri(URI& uri) const noe
 std::pair<fg::Error, fg::DataSource> fg::glTF::loadFileFromUri(URI& uri) const noexcept {
     auto path = directory / uri.path();
     std::error_code error;
+
+#if defined(__ANDROID__)
+    // Try to load external buffers from the APK. If they're not there, fall through to the file case
+    const auto android_result = loadFileFromApk(path);
+    if(android_result.first == Error::None) {
+        return android_result;
+    }
```

Comment:
`loadFileFromApk` does not check if `androidAssetManager` is `nullptr` or not. Perhaps this should be handled here, and skip the APK load when the manager is null.